scripts/electron/sandboxFix.js

```diff
@@ -0,0 +1,40 @@
+// copied and modified from https://github.com/gergof/electron-builder-sandbox-fix/blob/master/lib/index.js
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const fs = require('fs/promises')
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const path = require('path')
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const chalk = require('chalk')
+
+const log = (message, dotFormatting = chalk.blue) => {
+  console.log(`  ${dotFormatting('•')} ${message}`)
+}
+
+const afterPackHook = async (params) => {
+  if (params.electronPlatformName !== 'linux') {
+    // this fix is only required on linux
+    return
+  }
+
+  const executable = path.join(params.appOutDir, params.packager.executableName)
+
+  const loaderScript = `#!/usr/bin/env bash
+set -u
+SCRIPT_DIR="$( cd "$( dirname "\${BASH_SOURCE[0]}" )" && pwd )"
+exec "$SCRIPT_DIR/${params.packager.executableName}.bin" "--no-sandbox" "$@"
+`
```

Comment:
Can you try using `execa` instead of Bash script? Example of usage: https://github.com/Adamant-im/adamant-im/blob/dev/scripts/wallets.mjs#L1